README.md

```diff
@@ -1468,6 +1468,7 @@ Update Time, five active automations, webhooks.
   * [Icon Horse](https://icon.horse) – Get the highest resolution favicon for any website from our simple API.
   * [Iconoir](https://iconoir.com) – An open-source icons library with thousands of icons, supporting React, React Native, Flutter, Vue, Figma, and Framer.
   * [Icons8](https://icons8.com) — Icons, illustrations, photos, music, and design tools. Free Plan offers Limited formats in lower resolution. Link to Icons8 when you use our assets.
+  * [Infinite Table DataGrid](https://infinite-table.com) - The modern React DataGrid for enterprise applications. Free to use if you keep a footer link.
```

Comment:
So what are the SaaS features here?

Reply:
> This is a list of software (SaaS, PaaS, IaaS, etc.) and other offerings with free developer tiers

Sorry, this is not saas, but a useful UI component developers can use for free to build their saas product  - belongs to the category listed above - "other offerings with free developer tiers"

Reply:
You clicked it's a SaaS, this is a list for SaaS software?